Sources/FoundationMacros/CMakeLists.txt

```diff
@@ -12,10 +12,19 @@
 ##
 ##===----------------------------------------------------------------------===##
 
+cmake_minimum_required(VERSION 3.22)
+if(POLICY CMP0157)
+  cmake_policy(SET CMP0157 NEW)
+endif()
```

Comment:
While we're at it, can we also enable `CMP0156` as well? Thanks.

Reply:
Done